<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spectralLibraryView</class>
 <widget class="QMainWindow" name="spectralLibraryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>365</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Spectral Library</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qpsresources.qrc">
    <normaloff>:/qps/ui/icons/speclib.svg</normaloff>:/qps/ui/icons/speclib.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>false</bool>
      </property>
      <widget class="SpectralLibraryPlotWidget" name="mPlotWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QgsDualView" name="mDualView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="mProgressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mInfoLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="btnBarDualViewOptions">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnSpeclibProperties">
           <property name="text">
            <string>F</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../QGIS/images/images.qrc">
             <normaloff>:/images/themes/default/propertyicons/system.svg</normaloff>:/images/themes/default/propertyicons/system.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnRenderingView">
           <property name="text">
            <string>F</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../QGIS/images/images.qrc">
             <normaloff>:/images/themes/default/propertyicons/rendering.svg</normaloff>:/images/themes/default/propertyicons/rendering.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnFormView">
           <property name="text">
            <string>F</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../QGIS/images/images.qrc">
             <normaloff>:/images/themes/default/mActionFormView.svg</normaloff>:/images/themes/default/mActionFormView.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnTableView">
           <property name="text">
            <string>T</string>
           </property>
           <property name="icon">
            <iconset resource="../../../../QGIS/images/images.qrc">
             <normaloff>:/images/themes/default/mActionOpenTable.svg</normaloff>:/images/themes/default/mActionOpenTable.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBarAddProfiles">
   <property name="windowTitle">
    <string>Input / Output</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectProfilesFromMap"/>
   <addaction name="actionAddProfiles"/>
   <addaction name="actionSaveSpeclib"/>
  </widget>
  <widget class="QToolBar" name="toolBarFeatureSelections">
   <property name="windowTitle">
    <string>Selection Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectAll"/>
   <addaction name="actionInvertSelection"/>
   <addaction name="actionRemoveSelection"/>
   <addaction name="actionPanMapToSelectedRows"/>
   <addaction name="actionZoomMapToSelectedRows"/>
  </widget>
  <widget class="QToolBar" name="toolBarEditTools">
   <property name="windowTitle">
    <string>Edit Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReload"/>
   <addaction name="actionToggleEditing"/>
   <addaction name="actionCutSelectedRows"/>
   <addaction name="actionCopySelectedRows"/>
   <addaction name="actionPasteFeatures"/>
   <addaction name="actionDeleteSelected"/>
   <addaction name="actionSaveEdits"/>
   <addaction name="actionAddAttribute"/>
   <addaction name="actionRemoveAttribute"/>
  </widget>
  <action name="actionImportSpeclib">
   <property name="icon">
    <iconset resource="../qpsresources.qrc">
     <normaloff>:/qps/ui/icons/speclib_add.svg</normaloff>:/qps/ui/icons/speclib_add.svg</iconset>
   </property>
   <property name="text">
    <string>Import Spectral Library</string>
   </property>
   <property name="toolTip">
    <string>Import Spectral Library</string>
   </property>
  </action>
  <action name="actionSaveSpeclib">
   <property name="icon">
    <iconset resource="../qpsresources.qrc">
     <normaloff>:/qps/ui/icons/speclib_save.svg</normaloff>:/qps/ui/icons/speclib_save.svg</iconset>
   </property>
   <property name="text">
    <string>Save Spectral Library</string>
   </property>
   <property name="toolTip">
    <string>Save Profiles in Spectral Library</string>
   </property>
  </action>
  <action name="actionAddAttribute">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionNewAttribute.svg</normaloff>:/images/themes/default/mActionNewAttribute.svg</iconset>
   </property>
   <property name="text">
    <string>New field</string>
   </property>
   <property name="toolTip">
    <string>New field (Ctrl+W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionPasteFeatures">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionEditPaste.svg</normaloff>:/images/themes/default/mActionEditPaste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste features from clipboard</string>
   </property>
   <property name="toolTip">
    <string>Paste features from clipboard (Ctrl+V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRemoveAttribute">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionDeleteAttribute.svg</normaloff>:/images/themes/default/mActionDeleteAttribute.svg</iconset>
   </property>
   <property name="text">
    <string>Delete field</string>
   </property>
   <property name="toolTip">
    <string>Delete field (Ctrl+L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSaveEdits">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionSaveAllEdits.svg</normaloff>:/images/themes/default/mActionSaveAllEdits.svg</iconset>
   </property>
   <property name="text">
    <string>Save edits</string>
   </property>
   <property name="toolTip">
    <string>Save edits (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionToggleEditing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionToggleEditing.svg</normaloff>:/images/themes/default/mActionToggleEditing.svg</iconset>
   </property>
   <property name="text">
    <string>Toggle editing mode</string>
   </property>
   <property name="toolTip">
    <string>Toggle editing mode (Ctrl+E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSelectProfilesFromMap">
   <property name="icon">
    <iconset resource="../qpsresources.qrc">
     <normaloff>:/qps/ui/icons/profile_identify.svg</normaloff>:/qps/ui/icons/profile_identify.svg</iconset>
   </property>
   <property name="text">
    <string>Select Profiles from Map</string>
   </property>
   <property name="toolTip">
    <string>Select new profile from map</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionReload.svg</normaloff>:/images/themes/default/mActionReload.svg</iconset>
   </property>
   <property name="text">
    <string>Reload the table</string>
   </property>
   <property name="toolTip">
    <string>Reload the table</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionSelectAll.svg</normaloff>:/images/themes/default/mActionSelectAll.svg</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="toolTip">
    <string>Select all (Ctrl+A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionInvertSelection">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionInvertSelection.svg</normaloff>:/images/themes/default/mActionInvertSelection.svg</iconset>
   </property>
   <property name="text">
    <string>Invert selection</string>
   </property>
   <property name="toolTip">
    <string>Invert selection (Ctrl+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRemoveSelection">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionDeselectAll.svg</normaloff>:/images/themes/default/mActionDeselectAll.svg</iconset>
   </property>
   <property name="text">
    <string>Deselect all</string>
   </property>
   <property name="toolTip">
    <string>Deselect all (Ctrl+Shift+A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionPanMapToSelectedRows">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionPanToSelected.svg</normaloff>:/images/themes/default/mActionPanToSelected.svg</iconset>
   </property>
   <property name="text">
    <string>Pan map to the selected rows</string>
   </property>
   <property name="toolTip">
    <string>Pan map to the selected rows (Ctrl+P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoomMapToSelectedRows">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomToSelected.svg</normaloff>:/images/themes/default/mActionZoomToSelected.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom map to the selected rows</string>
   </property>
   <property name="toolTip">
    <string>Zoom map to the selected rows (Ctrl+J)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionDeleteSelected">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionDeleteSelected.svg</normaloff>:/images/themes/default/mActionDeleteSelected.svg</iconset>
   </property>
   <property name="text">
    <string>Delete selected features</string>
   </property>
   <property name="toolTip">
    <string>Delete selected features</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCutSelectedRows">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionEditCut.svg</normaloff>:/images/themes/default/mActionEditCut.svg</iconset>
   </property>
   <property name="text">
    <string>Cut selected rows to clipboard</string>
   </property>
   <property name="toolTip">
    <string>Cut selected rows to clipboard (Ctrl+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopySelectedRows">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionEditCopy.svg</normaloff>:/images/themes/default/mActionEditCopy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy selected rows to clipboard</string>
   </property>
   <property name="toolTip">
    <string>Copy selected rows to clipboard (Ctrl+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="optionAddCurrentProfilesAutomatically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../qpsresources.qrc">
     <normaloff>:/qps/ui/icons/profile_add_auto.svg</normaloff>:/qps/ui/icons/profile_add_auto.svg</iconset>
   </property>
   <property name="text">
    <string>Add profiles automatically</string>
   </property>
   <property name="toolTip">
    <string>Activate to add profiles automatically into the spectral library</string>
   </property>
  </action>
  <action name="actionFormView">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionFormView.svg</normaloff>:/images/themes/default/mActionFormView.svg</iconset>
   </property>
   <property name="text">
    <string>Form View</string>
   </property>
   <property name="toolTip">
    <string>Switch to form view</string>
   </property>
  </action>
  <action name="actionTableView">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionOpenTable.svg</normaloff>:/images/themes/default/mActionOpenTable.svg</iconset>
   </property>
   <property name="text">
    <string>Table View</string>
   </property>
   <property name="toolTip">
    <string>Switch to table view</string>
   </property>
  </action>
  <action name="actionAddProfiles">
   <property name="icon">
    <iconset resource="../qpsresources.qrc">
     <normaloff>:/qps/ui/icons/plus_green_icon.svg</normaloff>:/qps/ui/icons/plus_green_icon.svg</iconset>
   </property>
   <property name="text">
    <string>Add Profile(s)</string>
   </property>
   <property name="toolTip">
    <string>Adds curently overlaid profiles to the spectral library</string>
   </property>
  </action>
  <action name="actionDummy">
   <property name="text">
    <string>X</string>
   </property>
  </action>
  <action name="optionBlockProfiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qpsresources.qrc">
     <normaloff>:/qps/ui/icons/profile_dontreceive.svg</normaloff>:/qps/ui/icons/profile_dontreceive.svg</iconset>
   </property>
   <property name="text">
    <string>Don't add new profiles</string>
   </property>
   <property name="toolTip">
    <string>Block new profiles from being shown and added in this spectral library</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/propertyicons/system.svg</normaloff>:/images/themes/default/propertyicons/system.svg</iconset>
   </property>
   <property name="text">
    <string>Show Spectral Library Poperties</string>
   </property>
  </action>
  <action name="actionRenderingView">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/propertyicons/rendering.svg</normaloff>:/images/themes/default/propertyicons/rendering.svg</iconset>
   </property>
   <property name="text">
    <string>RenderingView</string>
   </property>
   <property name="toolTip">
    <string>Rendering view</string>
   </property>
  </action>
  <action name="actionImportVectorSource">
   <property name="icon">
    <iconset resource="../../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionAddOgrLayer.svg</normaloff>:/images/themes/default/mActionAddOgrLayer.svg</iconset>
   </property>
   <property name="text">
    <string>Import profiles from raster + vector source</string>
   </property>
   <property name="toolTip">
    <string>Import spectral profiles from a raster image based on vector geometries (Points).</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDualView</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectralLibraryPlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>qps.speclib.plotting</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qpsresources.qrc"/>
  <include location="../../../../QGIS/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
